backend/src/main/java/com/bakdata/conquery/models/forms/util/DateContext.java

```diff
@@ -270,8 +287,24 @@ public OptionalInt getAmountForAlignment(Alignment alignment){
 		QUARTER(CDateRange::getCoveredQuarters),
 		YEAR(CDateRange::getCoveredYears);
 
+		static {
+			NO_ALIGN.amountPerResolution = Map.of(Resolution.COMPLETE, 1);
```

Comment:
Auch hier mir leider nicht klar warum du das nicht im ctor machen kannst.

Frage mich gerade auch, ob das nicht sogar was wäre das schon in einer DatteTime Bibliothek kodiert ist.

Reply:
Ich glaube nicht, weil es unterschiedliche Definitionen für manches gibt. Im Finanzsektor ist ein Jahr manchmal nur 360 Tage lang.